spec/workers/sentry_worker_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe SentryWorker, type: :worker do
+  subject(:worker_subject) { described_class.new }
+
+  let(:event) { 'event' }
+
+  it 'adds job to a queue' do
+    described_class.perform_async(event)
+    expect(described_class).to have_enqueued_sidekiq_job(event)
+  end
+
+  it 'calls Raven.send_event with the correct arguments' do
+    allow(Raven).to receive(:send_event)
+    expect(Raven).to receive(:send_event).with(event)
```

Comment:
I suppose that this line should be after `#perform` call

Reply:
This is logical, but the test fails if up

Reply:
I mean next:
allow(Raven).to receive(:send_event)
worker.perform(event)
expect(Raven).to receive(:send_event).with(event)